apis/core.oam.dev/v1beta1/applicationrevision_types.go

```diff
@@ -31,6 +32,9 @@ type ApplicationRevisionSpec struct {
 	// Application records the snapshot of the created/modified Application
 	Application Application `json:"application"`
 
+	// ComponentManifests records the snapshot of resouces rendered from application components
+	ComponentManifests []RawComponentManifest `json:"componentManifests,omitempty"`
```

Comment:
why not re-use Components in L51

Reply:
And we already has a configmap record all resources after render

Reply:
If you want the information for convenience, you should also keep it in the configmap in L60